docs/using-radicle/fetching-and-merging.md

```diff
@@ -0,0 +1,142 @@
+to fetch and push changes from the server that hosts your project. In Radicle,
+these remotes point to the Git monorepo on your machine that stores all of your
+Radicle data. They allow you to fetch changes from your peers and push your
+changes. This means that fetching and pushing to remotes are _local first_
+actions. To read more about how Radicle is built on Git see our [**How it Works**][gi] section.
+
+To fetch the changes of a specific remote, run the `git fetch` command with
+their remote handle:
+
+
+```
+$ git fetch merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g
+remote: Enumerating objects: 7, done.
+remote: Counting objects: 100% (7/7), done.
+remote: Compressing objects: 100% (3/3), done.
+remote: Total 6 (delta 0), reused 6 (delta 0), pack-reused 0
+Unpacking objects: 100% (6/6), 1.50 KiB | 1.50 MiB/s, done.
+From rad://hwd1yre8wtxufyqnt8xzxt8ps3n7ww8qzpfmud5pibarumshawnpcrjzjme
+ * [new ref]         hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g/heads/main -> merle/main
+```
+
+A successful fetch will have a similar output to the example above and show that
+the reference is now located as a remote branch (e.g. `merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g/main`). We can
+confirm this is under a remote branch by entering the following command:
+
+
+```
+$ git branch -a
+main
+remotes/merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g/main
+remotes/rad/main
+```
+
+
+Once changes have been fetched, you can view the changes by getting a diff of
+your changes:
+
+```
+$ git diff merle/main 
+diff --git a/README.md b/README.md
+index b72e7d4..8c98464 100644
+--- a/README.md
++++ b/README.md
+@@ -1 +1 @@
+-Radicle is the next big thing!
++Radicle
+```
+
+If you are happy with these changes, you can merge them into your own branch:
+
+```
+$ git merge merle/main
+[main e6ee76d] Merge remote-tracking branch 'merle/main' into main
+```
+
+or for those of you who prefer rebasing:
+
+```
+$ git rebase merle/main
+[main 3e0e32d] Radicle Update
+ 1 file changed, 2 insertions(+)
+Successfully rebased and updated refs/heads/main.
+```
+
+This puts your working copy in a state this up-to-date with your contributor's
+changes. 
+
+You can publish these changes to the Radicle network by running `git push
+rad`(See [Pushing changes][pc]):
+
+```
+$ git push rad main
+Enumerating objects: 2, done.
+Counting objects: 100% (2/2), done.
+Delta compression using up to 8 threads
+Compressing objects: 100% (2/2), done.
+Writing objects: 100% (2/2), 1.72 KiB | 1.72 MiB/s, done.
+Total 2 (delta 0), reused 0 (delta 0), pack-reused 0
+To rad://hwd1yre8wtxufyqnt8xzxt8ps3n7ww8qzpfmud5pibarumshawnpcrjzjme.git
+   e6ee76d..57917bf  main -> main
+```
+
+Once you have published the changes from your working copy, you can view the
+changes you made in **Radicle Upstream**. If the changes were successfully
+published to the network, you should see the new commits on your project page.
```

Comment:
They are only published if you are online, and you can still see them when offline.

Reply:
Fixed this 👍